### PR TITLE
Enable windows CI on travis

### PR DESCRIPTION
**Pull request by feiskyer**
Travis now adds support for [Windows environment](https://docs.travis-ci.com/user/reference/windows/). This PR adds initial support for Windows building on travis and critest are disabled for Windows.

A few of following PRs will be added to fully support windows.

### CODE REVIEW

**Comment by feiskyer** [comment]
CI failed because of no enough space on disk. Seems this is a common issue of travis. refer https://travis-ci.community/t/there-is-not-enough-space-on-the-disk/338/3.

**Comment by PatrickLang** [comment]
cc @jterry75 - FYI this is coming

File: .travis.yml [hunk 1]
```diff
@@ -3,30 +3,57 @@ sudo: required
...
     - stage: Build
+      os: linux # Only need to run on Linux
       script:
-        - make release
+        -  make release
     - stage: Test
+      os: linux
       script:
         - make
+        - hack/install-docker.sh
```
Comment (feiskyer) [inline .travis.yml]: kubelet installation moved here, so that static check and build could be run more quickly.

File: hack/install-kubelet.ps1 [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+git clone -c core.symlinks=true https://github.com/kubernetes/kubernetes
+
+cd $kubernetespath
+$branch = [System.Environment]::GetEnvironmentVariable("TRAVIS_BRANCH")
+if ( ! "$branch".Equals("master") ) {
+  # We can do this because cri-tools have the same branch name with kubernetes.
+  git checkout "$branch"
+}
+
+# Build kubelet
```
Comment (feiskyer) [inline hack/install-kubelet.ps1]: Kubernetes makefile doesn't work on Windows, so here build kubelet directly by `go build`

**Pull request merged**